tutorials/notebooks/example_keras_pruning.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
```

Comment:
maybe already create this a a notebook and not a .py script?

Reply:
Sure. I will need to adapt it with ImageNet validation data, so I'll do it in a separate PR.